dmcontent/questions.py

```diff
@@ -634,10 +634,31 @@ def value(self):
 
     @property
     def answer_required(self):
+        """Checks all sub-questions and returns true if any questions which are required, still require answers.
+        Appropriately checks/ignores followup questions based on current answers."""
         if self.get('optional'):
             return False
 
-        return any(question.answer_required for question in self.questions)
+        lookup_question_by_id = {q.id: q for q in self.questions}
+        followup_questions = set()
+
+        # For each question that has followups, check the followup questions - if any require answers, the multiq does.
+        for question in self.questions:
+            answer = question.value if isinstance(question.value, list) else [question.value]
+
+            followups = question.get('followup', {})
+            if followups:
+                followup_questions.add(question.id)
+
+                for followup_id, values in followups.items():
+                    followup_questions.add(followup_id)
+
+                    if set(answer) & set(values) and lookup_question_by_id[followup_id].answer_required:
```

Comment:
I prefer set() being in the conditional; even though it gets executed and is slightly more expensive to run, I don't think we need to worry about optimising it out and having the two set() calls in such close proximity makes it obvious what's being compared. I care more about making things easy for the developer to understand than making things easy for the computer to execute, unless it's a significant trade-off or would become a bottleneck.